src/color/toColorString.js

```diff
@@ -6,38 +6,33 @@ import hsl from './hsl'
 import hsla from './hsla'
 import type { RgbColor, RgbaColor, HslColor, HslaColor } from '../types/color'
 
-const isRgb = (color): boolean =>
-  typeof color === 'object' &&
+const isRgb = (color: Object): boolean =>
   typeof color.red === 'number' &&
   typeof color.green === 'number' &&
   typeof color.blue === 'number' &&
-  // $FlowIgnoreNextLine not sure why this complains
-  typeof color.alpha !== 'number'
+  (typeof color.alpha !== 'number' || typeof color.alpha === 'undefined')
```

Comment:
yes, though I'm still working my way through this file, as it is the sole one left keeping us from having full flow support. It is proving...troublesome.